src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -1363,7 +1363,7 @@ private Wallet GenerateExtPubKeyOnlyWalletFile(string name, DateTimeOffset? crea
         }
 
         /// <summary>
-        /// Loads the wallet to be used by the manager.
+        /// Loads the wallet to be used by the manager if a wallet with this name has not already been loaded.
```

Comment:
Not related to this PR, but the trace log in this method is quite odd (`NOT_FOUND`). Maybe a typo?